app/controllers/wizard/steps/base_controller.rb

```diff
@@ -2,6 +2,18 @@ module Wizard
   module Steps
     class BaseController < ::ApplicationController
       default_form_builder GOVUKDesignSystemFormBuilder::FormBuilder
+
+      before_action :assign_commodity
```

Comment:
Would `fetch_commodity` be a better name here?

Reply:
We're not doing the fetch yet, though. We lazy load the response for when we need it in, say, a specific view or controller